components/modal.jsx

```diff
@@ -5,7 +5,9 @@ var ModalManagerMixin = require('../mixins/modal-manager');
 var Modal = React.createClass({
   mixins: [React.addons.PureRenderMixin, ModalManagerMixin],
   propTypes: {
-    modalTitle: React.PropTypes.string
+    modalTitle: React.PropTypes.string,
+    foldedStyle: React.PropTypes.bool,
```

Comment:
Should we just assume people will use `className` to pass in `"folded"` themselves? I'm a bit wary of adding a separate `propType` for each different style of modal that we might have, but I'm up for changing my mind...